pkg/storage/bloom/v1/builder.go

```diff
@@ -54,6 +57,7 @@ func NewBlockBuilder(opts BlockOptions, writer BlockWriter) (*BlockBuilder, erro
 
 	return &BlockBuilder{
 		opts:   opts,
+		writer: writer,
```

Comment:
As far as I see, the writer is only used to determine the size of the block after each population cycle.
Instead of adding the writer to the block builder, could we just use an int type that stores the current size?

Reply:
To get the Sizer you need access to the writer. I don't think we can just have an in in the builder, how would you set that int without access to the writer?